changelog.md

```diff
@@ -30,6 +30,21 @@ The route `/v2/schemas` has the following deprecation schedule:
 - 1.8.x - `/v2/schemas` is scheduled to be completely removed. If `--deprecated_features=json_schemas_resource` is
   still specified, Marathon will refuse to launch, with an error.
 
+### /v2/events
+
+The default response format of the `/v2/events` is marked as deprecated and will be switched to the `/v2/events?plan-format=light` in the first 1.7.x release. The following deprecation schedule is planned for this endpoint:
```

Comment:
shouldn't we do v3/events? I know it feels weird but ... we have kind of "versioned" API and still we are doing breaking change. I think it's better to decomission an endpoint than switch to a different format under the hood...

Just something to discuss...

Reply:
I thought about this. I feel like the current strategy is fine because:

- Fields are only removed, not renamed or changed type
- The event fields have never been formally specified
- If it causes issues, there is an option to add the problematic fields back while the clients remove their dependencies on these fields.

I really don't think it will be a problem. If it does, we have an safety hatch, and we can re-evaluate plans to fully remove in 1.8.x.

Reply:
ok, makes sense.

btw. removing fields is breaking change, only adding new fields is ok... :)

e.g. this guy will break, correct? 
https://github.com/gambol99/go-marathon/blob/master/events.go#L279

so we should at least pro-actively help them migrate...